src/fr/curriculum/index.html

```diff
@@ -8,7 +8,7 @@
 			<p>Les liens qui ne fonctionnent qu'au sein du réseau d'EDSC ou derrière le pare-feu du gouvernement du Canada sont marqués avec <i class="fas fa-external-link-square-alt" aria-hidden="true"> Icône</i><span class="wb-inv">Lien interne</span></p>
 		</div>
 	<!-- DataAjaxFragmentEnd -->
-</div>
+<section>
```

Comment:
I don't think the section element should be used here. The content after seem to belong under the h3 heading "Apprenants cibles"

Reply:
Removed all sections